### ISSUE TITLE
Docs not sync'ed with current release

### ISSUE THREAD

**Issue by MathewBiddle** [iobis/pyobis#116]
Docs state version 1.2.7: 
![image](https://user-images.githubusercontent.com/8480023/203616805-3205e80a-7f45-4902-9c6f-b91e43e7776a.png)

Looks like most recent release is 1.2.6: 
![image](https://user-images.githubusercontent.com/8480023/203616980-352d1c83-4807-40f9-aded-8ef1f60c8133.png)

Looks like 1.2.7 is a decent change from 1.2.6:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In [9], line 2
      1 from pyobis import dataset
----> 2 query = dataset.get(dataset_ids[0])
      3 query.execute() # execute the query
      4 # query.data # returns the data of the query
      5 # query.api_url # returns the OBIS API URL

AttributeError: module 'pyobis.dataset.dataset' has no attribute 'get'
```

Consider a larger version increment for these changes? 1.3.0??

**Comment by MathewBiddle** [comment]
Looks like that is the plan, things just got a little mixed up:
https://iobis.github.io/pyobis/changelog_link.html#id1

**Comment by ocefpaf** [comment]
@ayushanand18 this problem is not solved by #117. The issue here is b/c we are publishing every dev version when we merge PRs. I'll send a fix in a moment.
